jetbrains-core/src/software/aws/toolkits/jetbrains/services/sts/StsResources.kt

```diff
@@ -0,0 +1,14 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sts
+
+import software.amazon.awssdk.services.sts.StsClient
+import software.aws.toolkits.jetbrains.core.ClientBackedCachedResource
+
+object StsResources {
+    @Suppress("UsePropertyAccessSyntax")
+    val ACCOUNT = ClientBackedCachedResource(StsClient::class, "sts.account") {
```

Comment:
Should we make this have a super high expire time?

Reply:
made it 1 day